src/isIsogram.test.js

```diff
@@ -1,9 +1,33 @@
 'use strict';
 
-describe('isIsogram', () => {
-  const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
-    expect(isIsogram).toBeInstanceOf(Function);
-  });
+  const isIsogram = require('./isIsogram');
+
+
+it(`should be declared`, () => {
+  expect(isIsogram).toBeInstanceOf(Function);
+});
+
+it(`should return true for word without same letters`, () => {
```

Comment:
"Same letters" can be replaced by "repetitions" - but it is only a suggestion for the future.